### DISCUSSION TITLE
Matcher patterns for names with apostrophes.

### DISCUSSION THREAD

**Discussion by maraimm** [explosion/spaCy#8800] [Help: Coding & Implementations] [answered]
Hi, 

I am using the matcher to create patterns for people names.  Some of them names have apostrophe's, and they never get matched. When I checked the patterns, they seem to be surrounded by multiple double quotes 
```
{'ORTH': 'Murphy'}
"{'ORTH': ""O'Neill""}"
"{'ORTH': ""O'Shannon""}"
```
* Operating System: MacOS Catalina 
* Python Version Used: 3.9
* spaCy Version Used:  3.1.0

**Comment by polm** [comment, marked as answer]
Sorry you're having trouble with this. It seems to work without issue for me. 

```
import spacy 
from spacy.matcher import Matcher

nlp = spacy.load("en_core_web_sm")
matcher = Matcher(nlp.vocab)

pattern = [{"ORTH": "O'Leary"}]
matcher.add("APOSTROPHE", [pattern])

doc = nlp("O'Leary contains an apostrophe.")
for _, start, end in matcher(doc):
    print(doc[start:end])
```

Can you provide sample code that shows your problem? I'm not clear how you are checking the patterns that you get that kind of output. Note that in some cases Python changes the default way it shows quotes if strings contain apostrophes. 

It is possible that tokenizer settings might make names like `O'Shannon` three tokens, but that doesn't seem to be the case for the default English tokenizer.
